docs/installation/installation-using-docker-windows-10.md

```diff
@@ -0,0 +1,114 @@
+# Installation Using Docker on Windows 10
+**Expected installation time:** less than 1 hour.
+
+This guide covers building new projects based on Shopsys Framework.
+In case you want to contribute to the framework itself, you need to install the whole [shopsys/shopsys](https://github.com/shopsys/shopsys) monorepo.
+Take a look at the article about [Monorepo](../introduction/monorepo.md) for more information.
+
+## Supported systems
+- Windows 10 version 2004 and higher
+
+## System requirements
+- enabled Virtualization in BIOS
+
+## Installation of application requirement
+
+### Setup WSL 2 and download Debian
+!!! note
+    This guide covers installation using `Debian` distribution. Rest of guide might not be suitable for you, if you decide to select different distribution.
+
+In [Windows Subsystem for Linux Installation Guide for Windows 10](https://docs.microsoft.com/en-us/windows/wsl/install-win10#manual-installation-steps) follow steps 1 to 5.
+Select `Debian GNU/Linux` distribution in step 6.
+
+After installation open `Debian` application.
+You will be prompted to create new user.
+
+### Install Docker Desktop
+* Download [Docker Desktop](https://docs.docker.com/docker-for-windows/install/) and install it.
+* Docker Desktop requires at least 6.5 GB of memory, but this is only required to run `composer install` and `composer update` which would result in `Killed` status if not enough memory would be available (we recommend to set at least 2.5 GB RAM, 1 CPU and 4 GB Swap in `Docker -> Preferences… -> Resources -> ADVANCED`)
```

Comment:
I prefer to keep it there because of two reasons:
- it is not on same level as virtualization in Bios
- it is next step after installation so you do not forget about it

Reply:
But it is a requirement so I would expect it at the begining to know I do not need to try it on my low PCs. It would make more sense to me that way. It is not crucial, I will leave it open for BV to decide.

Reply:
I feel like it is not necessary,  windows OS is somehow needy in terms of resources anyway, soo I doubt very much developer on windows would be working on low end PC.

Reply:
I have removed that line